src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private Random random = new Random();
+
+    public String getRandomColor() {
+        return Colors.values()[random.nextInt(Colors.values().length)].toString();
```

Comment:
Instead of using toString() to get the String representation of an enum constant, use the name() method. This ensures consistency and avoids issues if toString() is overridden.